test/runtests.jl

```diff
@@ -38,7 +38,7 @@ device!(pick.dev)
 CuArrays.allowscalar(false)
 
 CuArrays.enable_timings()
-
+#=
```

Comment:
I think we need to support arguments to `runtests.jl` a la base for our sanity :-)

Reply:
oh my god derp

Reply:
ban me